anta/tests/interfaces.py

```diff
@@ -1342,3 +1342,81 @@ def _verify_interface_counters(self, interface_counters: dict[str, Any], interfa
                 self.result.is_failure(
                     f"{interface_failure_message_summary} - {counter['counter_name']} above threshold - Expected: {expected_counter_value} Actual: {counter_value}"
                 )
+
+
+class VerifyInterfacesTransceiverTemperature(AntaTest):
+    """Verify the interfaces transceiver temperature.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the temperature of all transceivers falls within the defined threshold.
+    * Failure: The test will fail if the temperature of any transceivers exceeds the defined threshold.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.interfaces:
+      - VerifyInterfacesTransceiverTemperature:
+          interfaces:
+            - Ethernet1/1
+            - Ethernet2/1
+          max_transceiver_temperature: 68
+    ```
+    """
+
+    categories: ClassVar[list[str]] = ["interfaces"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaCommand(command="show interfaces transceiver", revision=1)]
```

Comment:
This will hit the cache with the other test.
```suggestion
    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaCommand(command="show interfaces transceiver dom thresholds", revision=1)]
```

Reply:
updated thanks!!